src/translations/en/service.json

```
@@ -7,6 +7,7 @@
   "dialog.start.title": "Some points of attention",
   "error.cta": "Let us know!",
   "error.title": "We're sorry, an error occured",
+  "pageNotAccurate": "Page does not seem accurate? Try differently.",
```

If that is a call to action, it's good to mention it in the key 🙂 
```suggestion
  "pageNotAccurateCTA": "Page does not seem accurate? Try differently.",
```